### PR TITLE
feat: Add the SmartWearable filter to the Creations tab

### LINKED ISSUE #263: Add the smart items filter
Add the Smart Item filter into the `Creations` component's sidebar and add the API logic to query smart wearables as it's done in the Marketplace.
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBalQ1IiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--ee4faac3fd36eb4f22f2c0a933e57cb469e05e64/Screenshot%202023-10-02%20at%2023.31.35.png)

### PR DESCRIPTION
**Pull request by juanmahidalgo**
Closes #263 

image

### CODE REVIEW

File: src/components/Creations/Creations.tsx [hunk 1]
```diff
@@ -73,6 +74,12 @@ const Creations = (props: Props) => {
     },
     [getRaritiesQueryString, changeFilter]
   )
+  const handleSetSW = useCallback(
+    (isOnlySmart: boolean) => {
+      changeFilter('isWearableSmart', `${isOnlySmart}`)
```
Comment (LautaroPetaccio) [inline src/components/Creations/Creations.tsx:79]: Just as an alternative:
```suggestion
      changeFilter('isWearableSmart', isOnlySmart.toString())
```

File: src/components/Creations/Creations.tsx [hunk 2]
```diff
@@ -99,6 +106,11 @@ const Creations = (props: Props) => {
           <div className={styles.sidebar}>
             <CategoryFilter i18n={{ title: t('categories_menu.title') }} items={categories} value={category} onClick={handleSetCategory} />
             <RarityFilter rarities={rarities} onChange={handleSetRarity} />
+            <SmartWearableFilter
```
Comment (LautaroPetaccio) [inline src/components/Creations/Creations.tsx:109]: The `SmartWearableFilter` should be shown only when a wearable is being shown, would you mind adding a condition to show it under that circumstance? We have a function called `getCategoryName` that returns either `wearables` or `emotes` depending on the category.

File: src/modules/items/client.ts [hunk 3]
```diff
@@ -97,6 +97,10 @@ export class ItemsClient {
       }
     }
 
+    if (filters.isWearableSmart) {
```
Comment (LautaroPetaccio) [inline src/modules/items/client.ts:100]: Would you mind adding a test in the client for this filter?

**Review by LautaroPetaccio — COMMENTED** [review]
The PR looks great! 👍  I've left some minor comments about some tests and the filter.

File: src/components/Creations/Creations.tsx (hunk 2 above)
Comment (juanmahidalgo) [reply src/components/Creations/Creations.tsx]: Ohh good catch, sure, added!

File: src/modules/items/client.ts (hunk 3 above)
Comment (juanmahidalgo) [reply src/modules/items/client.ts:100]: Added!

**Review by LautaroPetaccio — APPROVED** [review]

**Pull request merged**